web/templates/template_landing.html

```diff
@@ -164,7 +164,7 @@ <h1 class="brand-heading">Green Analytics</h1>
 
 
   <h4>Accessing Dashboard</h4>
-  <p> <a href='#'>{{ context.dashboard }}/?token={{ context.site_key }}</a> ,
+  <p> <a href='#'>{{ context.dashboard }}?token={{ context.site_key }}</a> ,
```

Comment:
I don't know if there are side-effects when we set PROTOCOL=https in .env. If I'm not mistaken, it should work, though.